Content.Shared/Standing/StandingStateComponent.cs

```diff
@@ -20,8 +20,8 @@ public sealed class StandingStateComponent : Component
         ///     List of fixtures that had vault-impassable prior to an entity being downed. Required when re-adding the
         ///     collision mask.
         /// </summary>
-        [DataField("vaultImpassableFixtures")]
-        public List<string> VaultImpassableFixtures = new();
+        [DataField("highImpassableFixtures")]
+        public List<string> HighImpassableFixtures = new(); // TODO ShadowCommander: Figure out if this is needed anymore
```

Comment:
Not sure what this was for, but the usages don't work with the new humanoid collision mask.

Reply:
its so that players who are knocked down don't collide with high impassable, and it stores the fixtures that had `HighImpassable` so that it can properly readd them later when they stand back up